brut.apktool/apktool-lib/src/main/java/brut/androlib/Config.java

```diff
@@ -16,13 +16,15 @@
  */
 package brut.androlib;
 
+import com.android.tools.smali.baksmali.BaksmaliOptions;
+
 import brut.androlib.exceptions.AndrolibException;
 import brut.util.OSDetection;
 
 import java.io.File;
 import java.util.logging.Logger;
 
-public class Config {
+public class Config extends BaksmaliOptions {
```

Comment:
so you can still set the default options to override baksmali's defaults, this was only extended so that I could change the format of what gets annotated in the smali files, as well as remove the requirement to also provide a PREFIX to the argument on the command line. I didn't end up exposing any of the additional functions that inadvertently become available with this change, so to end users the only thing that would be different is the one additional command line option I added.